src/Uno.UI/UI/Xaml/UIElement.cs

```diff
@@ -822,6 +833,19 @@ internal string GetThemeSource(string propertyName)
 			}
 		}
 
+		private DependencyProperty GetDPByName(string propertyName) => GetDPByName(propertyName, this);
+
+		private static DependencyProperty GetDPByName(string propertyName, DependencyObject propertyOwner)
+		{
+			if (!propertyName.EndsWith("Property"))
```

Comment:
Can't you use this ?

https://github.com/unoplatform/uno/blob/a610f88c0f39f894c0879bc825ec863b1cae2f55/src/Uno.UI/UI/Xaml/DependencyProperty.cs#L274

Reply:
fixed